devservices/config.yml

```diff
@@ -0,0 +1,59 @@
+# Ignored by docker compose, used by devservices
+x-sentry-service-config:
+  version: 0.1
+  service_name: launchpad
+  dependencies:
+    redis:
+      description: Shared instance of redis used by sentry services
+      remote:
+        repo_name: sentry-shared-redis
+        branch: main
+        repo_link: https://github.com/getsentry/sentry-shared-redis.git
+    kafka:
+      description: Shared instance of kafka used by sentry services
+      remote:
+        repo_name: sentry-shared-kafka
+        branch: main
+        repo_link: https://github.com/getsentry/sentry-shared-kafka.git
+    launchpad:
+      description: Service that powers preprod artifact analysis
+  modes:
+    default: [redis, kafka]
+    containerized: [redis, kafka, launchpad]
+
+services:
+  launchpad:
+    image: ghcr.io/getsentry/launchpad:latest
+    ports:
+      - 127.0.0.1:1218:1218
+      - 127.0.0.1:1219:1219
+    command:
+      - devserver
+      - --${LAUNCHPAD_NO_WORKERS:+no-workers}
+    healthcheck:
+      test: curl -f http://localhost:1218/health_envoy
+      interval: 5s
+      timeout: 5s
+      retries: 3
+    environment:
+      PYTHONUNBUFFERED: 1
+      LAUNCHPAD_SETTINGS: docker
```

Comment:
A lot of these env variables are snuba specific, wondering if they could be removed at least for now until they should be added